southworkscom/botframework-solutions
# [TypeScript][Virtual Assistant Template] Add Skill folder

## Purpose
We added the `Skills` folder to maintain the consistency with C# template.

## Changes
- Add the `Skills` folder in the Virtual Assistant Template
- Update the Virtual Assistant Sample

## Testing Steps
1. Execute `yo botbuilder-assistant` and follow the prompts.
1. Verify that the `Skills` folder is copied to `resources`, that matches with the following image.

![image](https://user-images.githubusercontent.com/40918752/61083397-027ac180-a402-11e9-8e1a-313026fc960c.png)
